toolchain/sem_ir/formatter.cpp

```diff
@@ -32,8 +32,9 @@ class InstNamer {
   enum class ScopeId : int32_t {
     None = -1,
     File = 0,
-    Constants = 1,
-    FirstFunction = 2,
+    ImportRef = 1,
```

Comment:
I'm not seeing this used in the tests, am I misunderstanding how this works and it doesn't show up like `file` and such?